docs/rhc4tp-cluster.md

```diff
@@ -1,73 +1,77 @@
-Release pipeline publishes the images to public registries, which are hosted at RHC4TP clusters.
-To prepare the RHC4TP clusters to be used by the pipeline, for each of environments: 
+# RHC4TP OpenShift Cluster Setup
 
-0. Login to the chosen cluster
+The release pipeline publishes bundle images to registry.connect.redhat.com. The registry
+behind that address is the RHC4TP OpenShift cluster. The following steps must be performed
+to prepare the RHC4TP clusters for use by the pipeline. 
+
+1. Login to the chosen cluster
 
 1. Create a service account
 
-```bash
-oc create sa operator-pipelines -n default
-```
+    ```bash
+    oc create sa operator-pipelines -n default
+    ```
+
+1. Create a Kubeconfig for the service account. It should be stored in the repository Ansible Vault.
```

Comment:
```suggestion
3. Create a kubeconfig for the service account. It should be stored in the repository Ansible Vault.
```

Reply:
Please fix the numbers of the bullets accordingly :)

Reply:
See my other comment.